runtime/bindings/python/tests/test_inference_engine/test_core.py

```diff
@@ -139,14 +137,8 @@ def test_net_from_buffer_valid():
         xml = f.read()
     func = core.read_model(model=xml, weights=bin)
     ref_func = core.read_model(model=test_net_xml, weights=test_net_bin)
-    assert func.name == func.name
-    assert func.batch_size == ref_func.batch_size
-    ii_func = func.input_info
-    ii_func2 = ref_func.input_info
-    o_func = func.outputs
-    o_func2 = ref_func.outputs
-    assert ii_func.keys() == ii_func2.keys()
-    assert o_func.keys() == o_func2.keys()
+    assert func.get_parameters() == ref_func.get_parameters()
+    assert func.get_results() == ref_func.get_results()
```

Comment:
also  you may compare here 
```suggestion
    assert func.get_results() == ref_func.get_results()
    assert func.get_ordered_ops() == ref_func.get_ordered_ops()
```